### DISCUSSION TITLE
Where should I put syntax files?

### DISCUSSION THREAD

**Discussion by sravioli** [NvChad/NvChad#2123] [Q&A] [answered]
According to <https://neovim.io/doc/user/syntax.html> I should place them in `nvim/syntax` but that is not gitignored bu NvChad. Where should i put the syntax? Is it in `ftplugin`?

**Comment by siduck** [comment, marked as answer]
@sRavioli add a pr that adds nvim/syntax into gitignore. but why do you need the syntax files?

**Reply by sravioli** [reply]
I use plain text files for some of my uni courses and it would be helpful to have custom defined highlighting. Yes it's a pretty specific use case
